src/get-entries.js

```diff
@@ -1,6 +1,6 @@
 const core = require('@actions/core')
 
-const versionSeparator = /\n#{1,3}\s/
+const versionSeparator = '\n## '
```

Comment:
It gets the version number on three different title level (#, ##, ###). The downside is that if your Changelog is very unstructured it might not get the data correctly.

The more I think of the change, the more I'm open to revert it. Another way of handling it is to have better unit tests to ensure we still support correctly the main CHANGELOG formats.